generators/spa/templates/VueJS/common/Scripts/QuickSettings.js

```diff
@@ -1,32 +1,97 @@
-var <%= moduleName %>Settings = <%= moduleName %>Settings || {};
+var dnnspamodule = dnnspamodule || {};
 
-<%= moduleName %>.services = {}; // we need a service reference for each module
+dnnspamodule.quickSettings = function (root, moduleId) {
+    console.log(moduleId);
+    var utils = new common.Utils();
+    var alert = new common.Alert();
+    var parentSelector = "[id='" + root + "']";
+    // Setup your settings service endpoint
+    var service = {
+        baseUrl: "DesktopModules/<%= moduleName %>/API/",
+        framework: $.ServicesFramework(moduleId),
+        controller: "Settings"
+    }
 
-jQuery(function ($) {
-    <%= moduleName %>Settings.service.framework = $.ServicesFramework(0); // TODO
-    <%= moduleName %>Settings.service.baseUrl = <%= moduleName %>Settings.service.framework.getServiceRoot(<%= moduleName %>Settings.service.path) + "Settings/";
-});
+    var SaveSettings = function () {
 
-<%= moduleName %>Settings.InitApp = function (moduleid) {
-    var svc = {
-        moduleid: moduleid,
-        path: "<%= moduleName %>",
-        framework: $.ServicesFramework(moduleid)
+        var name = $('#Name').is(":checked");
+        var description = $('#Description').is(":checked");
+        var assignedUserId = $('#AssignedUserId').is(":checked");
+        var createdOnDate = $('#CreatedOnDate').is(":checked");
+        var itemId = $('#ItemId').is(":checked");
+
+        var deferred = $.Deferred();
+        var params = {
+            name: name,
+            description: description,
+            assignedUserId: assignedUserId,
+            createdOnDate: createdOnDate,
+            itemId: itemId
+        };
+
+        utils.get("POST", "save", service, params,
```

Comment:
@VAlmea I don't see it here, so it may be within the destination method... Are we passing in and checking for the services framework antiforgery tokens in this API call?

Reply:
@VAlmea You never responded to this question.

Reply:
Yes. Before sending the request we call the `service.framework.setModuleHeaders` method. And the controller actions have the `[ValidateAntiForgeryToken]` action filter